cmd/apps/gitea_app.go

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) arkade author(s) 2020. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package apps
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+
+	"github.com/alexellis/arkade/pkg/apps"
+	"github.com/alexellis/arkade/pkg/k8s"
+	"github.com/alexellis/arkade/pkg/types"
+
+	"github.com/alexellis/arkade/pkg"
+	"github.com/alexellis/arkade/pkg/config"
+	"github.com/alexellis/arkade/pkg/env"
+	"github.com/alexellis/arkade/pkg/helm"
+	"github.com/sethvargo/go-password/password"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallGitea() *cobra.Command {
+	var gitea = &cobra.Command{
+		Use:          "gitea",
+		Short:        "Install gitea",
+		Long:         `Install gitea`,
+		Example:      `  arkade install gitea`,
+		SilenceUsage: true,
+	}
+
+	gitea.Flags().Bool("update-repo", true, "Update the helm repo")
+	gitea.Flags().String("namespace", "default", "Kubernetes namespace for the application")
+	gitea.Flags().Bool("persistence", false, "Enable persistence")
+	gitea.Flags().String("admin-user", "gitea_admin", "Username of admin user")
```

Comment:
What about `user` and StringP to enable `-u` as a shortcut?

Reply:
updated